doc/reference/algorithms/community.rst

```diff
@@ -29,6 +29,7 @@ Modularity-based communities
    :toctree: generated/
 
    greedy_modularity_communities
+   _naive_greedy_modularity_communities
```

Comment:
Alternatively should this one be removed from `community.__all__`?

Reply:
Yeah -- this one has the underscore in front. It's doc_string simply says that this way of doing it is slow (O(N^4)) but it is easy to understand.  I'd call that a pedagogical function. We should either take off the underscore or take it out of ```__all__```. It's good to have some functions like that. How do we make them known without making them public?

Reply:
I think I will take off the leading underscore and make the function public. The docstring should be enough to keep people from using it. And it will still be found via the docs.